src/commands/Settings.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  AutocompleteInteraction,
+  EmbedBuilder,
+  InteractionType,
+  PermissionsBitField,
+  SlashCommandBuilder,
+  SlashCommandSubcommandBuilder,
+  type ChatInputCommandInteraction
+} from "discord.js";
+import { genColor } from "../utils/colorGen";
+import {
+  getSetting,
+  setSetting,
+  settingsDefinition,
+  settingsKeys
+} from "../utils/database/settings";
+import { errorEmbed } from "../utils/embeds/errorEmbed";
+
+export default class Settings {
+  data: SlashCommandBuilder;
+  constructor() {
+    this.data = new SlashCommandBuilder()
+      .setName("settings")
+      .setDescription("Configure Sokora to your liking.")
+      .setDefaultMemberPermissions(PermissionsBitField.Flags.Administrator);
+
+    settingsKeys.forEach(key => {
+      const subcommand = new SlashCommandSubcommandBuilder()
+        .setName(key)
+        .setDescription("This subcommand has no description.");
```

Comment:
We should add a way to get the description instead of having this for every subcommands

Reply:
i have an idea about that

Reply:
optix has fixed that

Reply:
this was resolved 👍